frontend/src/host_orchestrator/orchestrator/instancemanager.go

```diff
@@ -852,3 +856,18 @@ func downloadArtifactToFile(buildAPI BuildAPI, filename, artifactName, buildID,
 	downloadErr = buildAPI.DownloadArtifact(artifactName, buildID, target, f)
 	return downloadErr
 }
+
+// Acloud only works with the first instance: cvd-1
+func runAcloudSetup(runtimesRootDir string) {
```

Comment:
I prefer to not do this in the instance manager or any other component of the host orchestrator. Also, it's basically running a command as another user, so a shell script is a better place to put this.
For example, it's possible to create a symlink to a target that doesn't exists, so, if `runtimesRootDir` is known at boot time, you could add something to the script that starts the host-orchestrator to create the link. That script runs as root IIRC, so you wouldn't need to give `_cutf-operator` permissions to execute as another user.

Reply:
The caveat with this approach is user `vsoc-01` and `/home/vsoc-01` directory might not exist when the package is installed.

Reply:
the script could create the user, which would guarantee the directory's existence, but that may break the part that creates it today, so it's probably not worth it.

Reply:
agree.